status-im/status-react
# Feature fix/ttt settings

Splitted the final touches to the settings part of ttt to limit scope of the final ttt PR

- settings are stored in a manifest that is pointed at by the contract
- during login the contract is checked to fetch the settings

### Steps to test

On Ropsten on an account with some test eth, go to settings and set up tribute to talk, try recovering the account on another device (should fetch the settings), try changing network (shouldn't keep settings).

status: ready
